frontend/packages/dev-console/src/components/dropdown/ResourceDropdown.tsx

```diff
@@ -135,6 +136,10 @@ class ResourceDropdown extends React.Component<ResourceDropdownProps, State> {
     ) {
       this.onChange(allSelectorItem.allSelectorKey);
     }
+
+    if (_.isEmpty(sortedList) && this.props.onLoad) {
```

Comment:
This should only be called when `this.props.loaded === true`, no?
Also, deconstruct `onLoad` from `props` at the start of this function.

Reply:
Yes, that is already checked on line no. 78.

Reply:
What about only on first load?
`if (nextProps.loaded && !this.props.loaded)`

Reply:
Agree. 
Done.